cluster-state-service/internal/features/e2e_stream_instances.feature

```diff
@@ -4,5 +4,14 @@ Feature: Stream Instances
   Scenario: Get instance stream
     When I start streaming all instance events
     And I start 1 task in the ECS cluster
+    And I get instance where the task was started
     Then the stream instances response contains at least 1 instance
-    And the stream instances response contains the cluster where the task was started
\ No newline at end of file
+    And the stream instances response contains the instance where the task was started
+
+  Scenario: Get instance stream with past entity version
+    Given I start 1 task in the ECS cluster
+    And I get instance where the task was started
+    And I stop the 1 tasks in the ECS cluster
```

Comment:
Nit: task -> tasks.

Changing `^I stop the (\d+) tasks in the ECS cluster$` to `^I stop the (\d+) task(?:|s) in the ECS cluster$` [here](https://github.com/blox/blox/blob/dev/cluster-state-service/internal/features/e2etasksteps/list_tasks_steps.go#L109) should help.